.github/workflows/test.yml

```diff
@@ -13,29 +13,16 @@ jobs:
           - nightly-2023-06-04
     steps:
       - name: checkout
-        uses: actions/checkout@v2
+        uses: actions/checkout@v4
       - name: toolchain
-        uses: actions-rs/toolchain@v1
+        uses: dtolnay/rust-toolchain@master
         with:
-          profile: minimal
           toolchain: ${{ matrix.rust }}
-          override: true
       - name: check
-        uses: actions-rs/cargo@v1
-        with:
-          command: check
-          args: --all-features --all-targets
+        run: cargo check --all-features --all-targets
       - name: test/debug
-        uses: actions-rs/cargo@v1
-        with:
-          command: test
-          args: --all-features
+        run: cargo test --all-features
```

Comment:
We've found `nextest` very useful in debugging slow tests. Could be worth switching it over

Reply:
Good idea! I'll test it out (pun intended).